fcrepo-connector-file/pom.xml

```diff
@@ -21,6 +21,12 @@
       <groupId>javax.jcr</groupId>
       <artifactId>jcr</artifactId>
     </dependency>
+
+    <dependency>
+      <groupId>org.modeshape</groupId>
+      <artifactId>modeshape-jcr</artifactId>
+      <scope>provided</scope>
```

Comment:
Good point. That is a higher and better ideal. We should factor as much JCR as we can down through the modules to `kernel-impl`.